validate/validate.go

```diff
@@ -252,6 +277,7 @@ func validateXfam(value []byte, fixed1, fixed0 uint64) error {
 	if xfam&(^fixed0) != 0 {
 		return fmt.Errorf("unauthorized xfam 0x%x as xfamFixed0 0x%x bits are set", xfam, fixed0)
 	}
+	logger.V(1).Info("Successfully validated XFAM field")
```

Comment:
Please print xfam, fixed0 and fixed1 as part of V=2 logs

Reply:
Done